.cirrus.yml

```diff
@@ -152,6 +152,20 @@ task:
         - sleep 10s
         # Use GUI instead of daemon and only change the one in start only (Not to affect the test_unixsockets
         - sed -i '/if \[\[ \$1 == "start" \]\]; then/,/fi/ s/\$agent daemon -d/$agent gui >test.log 2>\&1 \& sleep 15\n pid=$!\necho $pid > "\/tmp\/$2.pid"/' electrum_nmc/electrum/tests/regtest/regtest.sh
+    - env:
+        ELECTRUM_MODE: "GUI"
+        QT_QPA_PLATFORM: "offscreen"
+        Binaries: "WINE" 
+      wine_script:
+        - dpkg --add-architecture i386
+        - apt-get update
+        - apt-get -y install  wine-development wine32-development
+        - VERSION=$(git describe --tags --dirty --always | cut -c 1-9)
+        - curl -o namecoin.exe https://api.cirrus-ci.com/v1/artifact/build/$CIRRUS_BUILD_ID/Windows%20build/binaries/contrib/build-wine/dist/electrum-nmc-$VERSION.exe
```

Comment:
Is there a reason why this is named `namecoin.exe`? I think `electrum-nmc.exe` might me more intuitive.

Reply:
(`namecoin.exe` sounds more like a Namecoin Core thing.)

Reply:
Alright!